README.md

```diff
@@ -66,14 +83,16 @@ for (var i = 0; i < samples.length; i += sampleBlockSize) {
   rightChunk = right.subarray(i, i + sampleBlockSize);
   var mp3buf = mp3encoder.encodeBuffer(leftChunk, rightChunk);
   if (mp3buf.length > 0) {
-    //TODO: write your mp3 here
+    mp3Data.push(mp3buf);
   }
 }
 var mp3buf = mp3encoder.flush();   //finish writing mp3
 
 if (mp3buf.length > 0) {
-    //TODO: finish writing your mp3 here
+    mp3Data.push(mp3buf);
 }
+
+console.log(mp3Data);
```

Comment:
@rehoehle merging the pull request
it would be great if someone (you?) added a complete example of in browser record audio plus save to mp3 file